File: public/server/go/src/usr/testBaseCount.go
package usr

import (
	"GoPluginMagicModule/src/helper"
	"strconv"

	"github.com/lockval/go2plugin"
)

func (e export) Export_testBaseCount(input *go2plugin.Input) map[string]any {
	input.GetSubVal(input.UID, "mBase", "Count")
	input.GetAndLock()

	c := helper.GetResp(input, input.UID, "mBase", "Count")

	ci, _ := strconv.Atoi(c)
	ci++
	c = strconv.Itoa(ci)

	input.PutSubVal(input.UID, "mBase", "Count", c)
	input.PutAndUnlock()

	return map[string]any{"Hello": "Go"}
}

File: public/server/javascript/src/usr/testBaseCount.ts
import { Dict, DBOperate } from "../libs/lockvalserver";

export function main(input: DBOperate<any>) {
  input.GetSubVal(input.UID, "mBase", "Count");
  input.GetAndLock();

  let c = input.GetResp.IDKey[input.UID]?.KeySub["mBase"]?.SubVal["Count"];

  let ci = Number(c);
  ci++;
  c = ci.toString();

  input.PutSubVal(input.UID, "mBase", "Count", c);
  input.PutAndUnlock();

  return {};
}
